scripts/game.js

```diff
@@ -1,96 +1,91 @@
 let userName = "Bob";
-    let botOption = "";
-    let userOption = "";
-    let result = "";
-    let resultText = "";
-    
-    const displayElement = document.getElementById("display");
+let botOption = "";
+let userOption = "";
+let result = "";
+let resultText = "";
 
-    function enterName() {
-        userName = document.getElementById("user").value;
-    }
-    
-    function game(option) {
-        userOption = option;
-        bot();
-        outcome();
-        getResult();
-        drawUI();
-    }
-    
-    function getResult() {
-        if (result === "user") {
-            resultText =`${userName} Wins!`;
-        }
-        else if (result === "bot") {
-            resultText = `${userName} Loses.`;
-        }
-        else {
-            resultText = `It's a tie.`;
-        }
-    }
-    
-    function drawUI() {
-        displayElement.innerHTML = "";
-        
-        let movesElement = document.createElement("p");
-        movesElement.innerHTML = `${userName} picked ${userOption} and BOT picked ${botOption}`;
-        
-        let resultElement = document.createElement("p");
-        resultElement.innerHTML = resultText;
-        
-        displayElement.appendChild(movesElement);
-        displayElement.appendChild(resultElement);   
-    }
+function enterName() {
+  userName = document.getElementById("user").value;
+}
+
+function game(option) {
+  userOption = option;
+  bot();
+  outcome();
+  getResult();
+  drawUIGame();
+}
+
+function getResult() {
+  if (result === "user") {
+    resultText = `${userName} Wins!`;
+  } else if (result === "bot") {
+    resultText = `${userName} Loses.`;
+  } else {
+    resultText = `It's a tie.`;
+  }
+}
 
-    function bot() {
-        let option = Math.random();
-        if (option <= 0.333) {
-            botOption = "Rock";
-        } else if (option <= 0.666) {
-            botOption = "Paper";
-        } else {
-            botOption = "Scissors";
-        }
+function drawUIGame() { 
```

Comment:
you have `drawUI` in the other js file. There is a naming conflict, so I had to rename this one.